google/cloud/storage/internal/get_object_metadata_request_test.cc

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/storage/internal/get_object_metadata_request.h"
+#include <gmock/gmock.h>
+
+namespace google {
+namespace cloud {
+namespace storage {
+inline namespace STORAGE_CLIENT_NS {
+namespace internal {
+namespace {
+using ::testing::HasSubstr;
+using ::testing::Not;
+
+TEST(GetObjectMetadataRequestTest, OStream) {
+  GetObjectMetadataRequest request("my-bucket", "my-object");
+  request.set_multiple_parameters(Generation(1), IfMetaGenerationMatch(3));
+  std::ostringstream os;
+  os << request;
```

Comment:
Same nit as in the other review -- call os.str() once, save the value in a variable.